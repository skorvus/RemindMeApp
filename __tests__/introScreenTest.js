import 'react-native';
import React from 'react';
//
import introscreen from '../src/screens/introscreen';

//
import renderer from 'react-test-renderer';

describe('IntroScreen Component', () => {
  it('should run a test to check a number', ()=>{
    const IntroScreenComponent = renderer.create(<introscreen/>).toJSON();
    expect(IntroScreenComponent).toMatchSnapshot();
  });
});
