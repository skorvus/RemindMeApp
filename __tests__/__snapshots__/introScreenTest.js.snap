// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroScreen Component should run a test to check a number 1`] = `<introscreen />`;
